apps/web/assets/index.ts

```diff
@@ -35,5 +36,6 @@ export const grassBrown = grassBrown_;
 export const walletIcon = walletIcon_;
 export const poolGrassBlue = poolGrassBlue_;
 export const commImg = commImg_;
+export const proposalImg = proposalImg_;
```

Comment:
We we dont use direct export ? 
would be:
> export { default } from "./ProposalImage.png";

Reply:
idk why we have this structure to export images and svgs really

Reply:
Yeah I think there is better way but we can tackle it later.